lib/includes.js

```diff
@@ -144,12 +144,25 @@ module.exports = function (adapter, schemas) {
             fetchedIds[type] = fetchedIds[type].concat(remainingIds);
 
             if (!inclusionDescriptor.def.baseUri) {
-                return adapter.find(type, {query: {filter: {id: remainingIds.join(',')}}});
+                return new Promise(function (resolve, reject) {
+                    const url = '/' + type + '?filter[id]=' + remainingIds.join(',');
+                    server.inject({method: 'get', url: url, headers: req.headers}, function (res) {
+                        if (res.statusCode !== 200) {
+                            reject(new Error(JSON.stringify(res.result)))
```

Comment:
Would this not return a error string for a certain property only? We would want all the request to fail and be given a 400 saying ```The request you made can not be completed as one or more of the resources you asked for can't be retrieved.``` + maybe the name of the resources?